### PR TITLE
241 rse tasks and responsiblities section

### PR DESCRIPTION
**Pull request by mhagdorn**
I've put a brief paragraph on the top 5 in the intro of Section 4 (Foundational RSE competencies). I have turned the bullet points in 4.4 (RSE tasks and responsibilities) into paragraphs. I have also rearranged them to follow some order.

### CODE REVIEW

File: competencies.md [hunk 1]
```diff
@@ -582,72 +589,87 @@ but at the same time to be able to convey their knowledge and experience to expe
...
+technological advancements. Additionally, proper \ac{SE} skills often require
+knowledge of \gls{TEAM}, and \gls{PM}. Today this means effective use of
+\acp{IDE}, \gls{static-analysis} tools, \glspl{design-pattern} and documentation
+(for oneself and others).
+
+Software development also includes testing. This task is a manifestation of the
+\ac{SE} competencies of \gls{DOCBB} and \gls{MOD} since a model of the software
+is required in order to write good tests that facilitate understanding and
+documentation. Today this encompasses the knowledge of testing frameworks as
+well as continuous integration and \ac{CICD} practices. Additionally to being
```
Comment (jpthiele) [inline competencies.md:610]: The Additionally part reads a bit wrong. 
My gut feeling says it should be 'In addition to being tested,...'

**Review by jpthiele — APPROVED** [review]

**Review by jngrad — APPROVED** [review]

File: competencies.md [hunk 2]
```diff
@@ -582,72 +589,87 @@ but at the same time to be able to convey their knowledge and experience to expe
...
+tested, software should also be reproducible. Projects like ReproHack
+[@ReproHack] can greatly help in fostering that competency.
+
+Part of the \ac{FAIR} principles is to make software findable and reusable. The
+RSE needs to be able to decide when and why to keep a repository private. This
+decision requires knowledge in the \gls{RC}, to understand when it makes sense
+to open up or close down a repository. The \gls{USERS}, \gls{TEAM} and sometimes
+\gls{SP} skills are required to make this decision. Furthermore, knowledge of
+the practices and contractual regulations of the RSE's institution are also
```
Comment (tobyhodges) [inline competencies.md:619]:
```suggestion
the practices and contractual regulations of the RSE's institution is also
```

File: competencies.md [hunk 3]
```diff
@@ -582,72 +589,87 @@ but at the same time to be able to convey their knowledge and experience to expe
...
+presence on social media. The other \gls{TEAM}-related aspect is the embedding
+of RSE graduates into the community of RSEs, sharing the same set of values and
+competencies. We envision our RSE graduates to be a part of a strong network of
+other RSEs, tool-related communities, and the classical domain communities,
+making them more effective at supporting research. These networks are a lifelong
+manifestation where RSEs work to provide an inclusive environment for their
+peers and provide opportunities for life-long learning.
+
+RSEs are also mentoring colleagues. This necessitates giving good advice that
+fits to a projects stage in its life-cycle, thereby requiring knowledge of
```
Comment (tobyhodges) [inline competencies.md:661]:
```suggestion
fits to a project's stage in its life-cycle, thereby requiring knowledge of
```

File: competencies.md [hunk 4]
```diff
@@ -582,72 +589,87 @@ but at the same time to be able to convey their knowledge and experience to expe
...
+(\gls{SWLC}), and its context in its research domain and thus (\gls{RC}).
+Research software often starts out as a tool to answer a personal research
+question and becomes more important when other researchers rely on it. Some
```
Comment (tobyhodges) [inline competencies.md:664]: I ❤️ this sentence.

File: competencies.md [hunk 5]
```diff
@@ -582,72 +589,87 @@ but at the same time to be able to convey their knowledge and experience to expe
...
+research software might even be used to deal with critical questions such as
+weather forecasting or medical diagnosis. To formalise the process of giving
+good advice a classification of software is commonly used [@Wang2012;
+@Schlauch2018b] where research software can move from one class to another
+during its life-cycle. @Schlauch2018b classify applications based on their scope
```
Comment (tobyhodges) [inline competencies.md:669]:
```suggestion
weather forecasting or medical diagnosis. A classification of software is commonly used
to formalise the process of giving good advice [@Wang2012;
@Schlauch2018b] where research software can move from one class to another
during its life-cycle. @Schlauch2018b classify applications based on their scope
```

File: competencies.md [hunk 6]
```diff
@@ -582,72 +589,87 @@ but at the same time to be able to convey their knowledge and experience to expe
...
-  These are complemented by the development of new tools and methods
-  for providing and working with software metadata, such as
-  the Citation File Format project [@druskat_cff_2021] and HERMES [@druskat_hermes_2022].
-  Other efforts focus on Software Management Plans (e.g., [@alves_elixir_2021; @martinez_ortiz_practical_2022])
-  which could be helpful for RSEs at early stages (i.e., with not much experience of project management).
-  They give quick hints on what to look for regarding basic management for research
-  software (including information on e.g., licenses, releases, publication, citation)
-  together with some ongoing work on corresponding metadata [@giraldo_metadata_2023].
-  Metadata can also be used actively during and within a research project to inform the decision making processes [@Bird2016].
+RSE can be found.
```
Comment (tobyhodges) [inline competencies.md:596]:
```suggestion
RSE can be found.
We describe here some examples of the competencies applied in combination to
the set of current common tasks and challenges for RSEs identified during the Paderborn workshop.
```

**Review by tobyhodges — COMMENTED** [review]
A small number of minor language improvements, plus one sentence for framing, referring back to the Paderborn workshop outcomes again.

**Comment by mhagdorn** [comment]
Thanks @tobyhodges for the suggestions - I have incorporated them all

**Review by CaptainSifff — APPROVED** [review]

**Pull request merged**